parlai/crowdsourcing/tasks/fast_acute/analysis.py

```diff
@@ -648,6 +654,66 @@ def save_results(self, path: str = None):
                 f.write(compiled_text)
 
 
+class MultiRunAcuteAnalyzer(AcuteAnalyzer):
+    """
```

Comment:
Given that this would be really useful, especially we might be collecting acute eval sequentially. Do you think it is necessary to enforce that across different runs they must use the exact same pairing files. (people must be alerted that pairing files will be overriden if they launch multiple runs using the same command).

Reply:
Hm, not necessarily - for instance, run 1 might use models A and B and run 2 might use models A,B, and C; the pairings filepath will be different but we may still want to combine the results from the two runs